Modify EL batching to doc-wise streaming approach (#12367)
explosion/spaCy: website/docs/api/kb.mdx

## KnowledgeBase.get_candidates {id="get_candidates",tag="method"}

Given textual mentions for an arbitrary number of documents as input, retrieve a
list of candidate entities of type [`Candidate`](/api/kb#candidate) for each
mention. The [`EntityLinker`](/api/entitylinker) component passes a generator

> yielding all mentions to retreive candidates for as
> [`SpanGroup`](/api/spangroup)) per document. The decision of how to batch

Reviewer: ```suggestion
that yields mentions as [`SpanGroup`](/api/spangroup))s per document. 
The decision of how to batch
```
